tartiflette/types/enum.py

```diff
@@ -17,16 +17,17 @@ class GraphQLEnumValue:
     def __init__(self, value: Any = None, name: Optional[str]=None, description: Optional[str]=None):
         self.value = value
         self.name = str(name)
+        if not name:
```

Comment:
also don't do str(name) if name is none :)

Reply:
Enum has to be updated / reworked anyway :x